### PR TITLE
Add emoji support

### PR DESCRIPTION
**Pull request by Geometer1729**
I realized it needs to demojify before it checks if the bot was triggered by a message

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -26,6 +26,20 @@ function sendDirect(source, message) {
...
+function demojify(w){
+	for(i in subs){
+		s=subs[i]
+		l=s[0]
+		r=s[1]
+		w=w.replace(new RegExp(l,'g'),r)
```
Comment (Grissess) [inline index.js]: Not that this is a probable hot path, but for the sake of RegExp compilers and GC everywhere, do you think you could cache the construction of these RE objects somewhere outside of `demojify`?

File: index.js [hunk 2]
```diff
@@ -26,6 +26,20 @@ function sendDirect(source, message) {
...
+	}
+	w=w.replace(/   /g,' ')
```
Comment (Grissess) [inline index.js:39]: In theory, same here, not that it matters too much in these cases

**Review by Grissess — CHANGES_REQUESTED** [review]
lgtm minus some little perfomance nits

File: index.js (hunk 1 above)
Comment (Geometer1729) [reply index.js]: Would 
```
for(i in subs)
	subs[i][0]=RegExp(subs[i][0],'g')
```
and then using those objects in demojify accomplish this?

**Review by Grissess — COMMENTED** [review]
that'd do it :D

**Pull request merged**